tidy3d/components/base_sim/simulation.py

```diff
@@ -102,6 +107,14 @@ class AbstractSimulation(Box, ABC):
         description="String specifying the front end version number.",
     )
 
+    plot_units: Optional[LengthUnit] = pd.Field(
```

Comment:
Probably `plot_length_units` is clearer in case we have other units in the plot, e.g. time.